util/microservice-sdk/js/lib/configManager/KeyTransformer.js

```diff
@@ -0,0 +1,18 @@
+const transform = require('lodash/transform');
+
+/**
+ * Transforms all the keys in a object according to a transformation function.
+ *
+ * @param {*} obj
+ * @param {(value: string) => string} mapFunction function to be applied to all keys in the object
+ */
```

Comment:
The return of the function remains to be documented...
(I'm just kidding) :neckbeard: